File: Programming_Task_03_Bhakti_Parhad/Python_Files/number_analysis.py
n = int(input("Enter N: "))

total = 0
even = 0
odd = 0

for i in range(1, n + 1):
    total += i

    if i % 2 == 0:
        even += 1
    else:
        odd += 1

print("Sum =", total)
print("Even Numbers =", even)
print("Odd Numbers =", odd)

File: Programming_Task_03_Bhakti_Parhad/C_Files/number_analysis.c
#include <stdio.h>

int main() {
    int n, i;
    int total = 0, even = 0, odd = 0;

    printf("Enter N: ");
    scanf("%d", &n);

    for(i = 1; i <= n; i++) {
        total += i;

        if(i % 2 == 0)
            even++;
        else
            odd++;
    }

    printf("Sum = %d\n", total);
    printf("Even Numbers = %d\n", even);
    printf("Odd Numbers = %d\n", odd);

    return 0;
}